src/components/Requirements.vue

```diff
@@ -21,7 +21,7 @@
           <div :style="{'border-bottom': minor.display ? `2px solid #${reqGroupColorMap[req.group][0]}` : ''}"
             @click="activateMinor(id)" class="major-title" v-for="(minor, id) in minors" :key="minor.id">
             <p :style="{'font-weight': minor.display ? '500' : '', 'color' : minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}"  class="minor-title-top">{{minor.minorFN}}</p>
-            <p :style="{'color': minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}" class="minor-title-bottom">({{user.collegeFN}})</p> <!-- Change for multiple colleges -->
```

Comment:
What was this for again?

Reply:
@einc explained it in #118 - but it used to display the name of the college under the minor, even though minors are not dependent on what college they are in. Hiding it for now.

Reply:
Gotcha